app/weather_parser.py

```diff
@@ -0,0 +1,46 @@
+import forecastio
+import json
+import requests
+from config import FORECAST_API_KEY, GOOGLE_API_KEY
+
+
+forecast_api_key = FORECAST_API_KEY
+geocode_api_key = GOOGLE_API_KEY
+geocode_url = 'https://maps.googleapis.com/maps/api/geocode/json?'
```

Comment:
you promised to put the Google stuff in a different API. Don't break your promises @admiral96 , get this stuff out of here.